elements/pfe-icon/test/pfe-icon_config-test-1.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <script src="/components/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src="/components/web-component-tester/browser.js"></script>
+  <script>
+    window.PfeConfig = {
+      PfeIconSets: [
+        {
+          name: "rh",
+          path: "https://static.redhat.com/libs/redhat/rh-iconfont/4.2.0/svg"
```

Comment:
I think we want to avoid committing examples pointing to static.redhat.com in the upstream project.

Reply:
👍